README.md

```diff
@@ -185,10 +183,12 @@ Let's use `babel` as an example.
 - Lerna checks if each dependency is also part of the Lerna repo.
   - In this example, `babel-generator` is a dependency, while `source-map` is not.
   - `source-map` is `npm install`ed like normal.
-- `babel-core/node_modules/babel-generator` is replaced with two files:
-  - A `package.json` with keys `name` and `version`
-  - An `index.js` file with the contents `module.exports = require("relative-path-to-babel-generator-in-the-lerna-repo")`
-- This links the `babel-generator` package in `node_modules` with the actual `babel-generator` files.
+- `packages/babel-core/node_modules/babel-generator` symlinks to `packages/babel-generator`
+- This allows nested directory imports
+
+**Note:** Circular dependencies result in circular symlinks which *may* impact your editor/IDE.
+
+[Webstorm](https://www.jetbrains.com/webstorm/) locks up when circular symlinks are present. To prevent this, add `node_modules` to the list of ignored files and folders in `Preferences | Editor | File Types | Ignored files and folders`.  
```

Comment:
In IDEA it's `Ignore files and folders` (no **d** in Ignore), perhaps the same in WebStorm?